app/models/models.go

```diff
@@ -153,6 +153,62 @@ func (r *ListQueueRequest) SetAttributesFromForm(values url.Values) {
 	r.QueueNamePrefix = values.Get("QueueNamePrefix")
 }
 
+/*** Send Message Request */
+
+func NewSendMessageRequest() *SendMessageRequest {
+	return &SendMessageRequest{
+		MessageAttributes:       make(map[string]MessageAttributeValue),
+		MessageSystemAttributes: make(map[string]MessageAttributeValue),
+	}
+}
+
+type SendMessageRequest struct {
+	DelaySeconds            int                              `json:"Del1aySeconds" schema:"DelaySeconds"`
+	MessageAttributes       map[string]MessageAttributeValue `json:"MessageAttributes" schema:"MessageAttributes"`
+	MessageBody             string                           `json:"MessageBody" schema:"MessageBody"`
+	MessageDeduplicationId  string                           `json:"MessageDeduplicationId" schema:"MessageDeduplicationId"`
+	MessageGroupId          string                           `json:"MessageGroupId" schema:"MessageGroupId"`
+	MessageSystemAttributes map[string]MessageAttributeValue `json:"MessageSystemAttributes" schema:"MessageSystemAttributes"`
+	QueueUrl                string                           `json:"QueueUrl" schema:"QueueUrl"`
+}
+type MessageAttributeValue struct {
+	BinaryListValues []string `json:"BinaryListValues"` // goaws does not supported yet
+	BinaryValue      string   `json:"BinaryValue"`
+	DataType         string   `json:"DataType"`
+	StringListValues []string `json:"StringListValues"` // goaws does not supported yet
+	StringValue      string   `json:"StringValue"`
+}
+
+func (r *SendMessageRequest) SetAttributesFromForm(values url.Values) {
```

Comment:
Looks like this could use a test?  Something like below?

https://github.com/Admiral-Piett/goaws/blob/feature-aws-json/app/models/models_test.go#L39-L85

Reply:
(Now in my TODO)

Reply:
Added!